common/socks5/requests/src/response.rs

```diff
@@ -318,6 +352,12 @@ impl ConnectionError {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub enum QueryResponse {
+    OpenProxy(bool),
```

Comment:
it's just a personal bias, but I'm not the biggest fan of tuples in this context, imo named fields would be better so once/if we add more fields, we'd still know what corresponds to what